src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+#web server
+server.port=8080
+server.contextPath=/api
+
+#Application specific
+authentication.oauth.clientid=webclient
+authentication.oauth.secret=secret
+authentication.oauth.tokenValidityInSeconds=1800
```

Comment:
This means if a user stays on the site for more than 5 min, they will be logged out. 30 min is very reasonable.